lib/github.rb

```diff
@@ -38,7 +38,10 @@ def self.freeze
   # :nocov:
 
   def self.runner_labels
-    @runner_labels ||= YAML.load_file("config/github_runner_labels.yml").to_h { [_1["name"], _1] }
+    @runner_labels ||= begin
+      labels = YAML.load_file("config/github_runner_labels.yml").to_h { [_1["name"], _1] }
+      labels.transform_values { |v| (a = v["alias_for"]) ? labels[a] : v }
```

Comment:
we probably have a problem if a label is an alias pointing to another alias. should we add a test to make sure after transformation no label exists with `alias_for` anymore?

Reply:
Good idea. It also helpful if we make a typo in alias. I added following test.

```ruby
  it "can map all aliases to actual tag" do
    expect(described_class.runner_labels.values).to be_all
  end
```